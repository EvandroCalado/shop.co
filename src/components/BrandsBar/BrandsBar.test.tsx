import { render, screen } from '@testing-library/react';
import { BrandsBar } from '.';

describe('<BrandsBar />', () => {
  it('shroud render with default values', () => {
    render(<BrandsBar />);

    expect(screen.getByAltText('versace')).toBeInTheDocument();
    expect(screen.getByAltText('zara')).toBeInTheDocument();
    expect(screen.getByAltText('gucci')).toBeInTheDocument();
    expect(screen.getByAltText('prada')).toBeInTheDocument();
    expect(screen.getByAltText('calvin')).toBeInTheDocument();
  });

  it('shroud render a snapshot', () => {
    const { container } = render(<BrandsBar />);

    expect(container).toMatchSnapshot();
  });
});
