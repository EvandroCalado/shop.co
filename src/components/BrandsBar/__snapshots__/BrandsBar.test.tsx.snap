// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<BrandsBar /> > shroud render a snapshot 1`] = `
<div>
  <section
    class="flex w-full flex-wrap items-center justify-around gap-4 bg-black/95 p-4 md:h-32"
  >
    <div
      class="relative h-[33px] w-[166px]"
    >
      <img
        alt="versace"
        data-nimg="fill"
        decoding="async"
        fetchpriority="high"
        sizes="166px"
        src="/_next/image?url=%2Fversace.png&w=3840&q=75"
        srcset="/_next/image?url=%2Fversace.png&w=16&q=75 16w, /_next/image?url=%2Fversace.png&w=32&q=75 32w, /_next/image?url=%2Fversace.png&w=48&q=75 48w, /_next/image?url=%2Fversace.png&w=64&q=75 64w, /_next/image?url=%2Fversace.png&w=96&q=75 96w, /_next/image?url=%2Fversace.png&w=128&q=75 128w, /_next/image?url=%2Fversace.png&w=256&q=75 256w, /_next/image?url=%2Fversace.png&w=384&q=75 384w, /_next/image?url=%2Fversace.png&w=640&q=75 640w, /_next/image?url=%2Fversace.png&w=750&q=75 750w, /_next/image?url=%2Fversace.png&w=828&q=75 828w, /_next/image?url=%2Fversace.png&w=1080&q=75 1080w, /_next/image?url=%2Fversace.png&w=1200&q=75 1200w, /_next/image?url=%2Fversace.png&w=1920&q=75 1920w, /_next/image?url=%2Fversace.png&w=2048&q=75 2048w, /_next/image?url=%2Fversace.png&w=3840&q=75 3840w"
        style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
      />
    </div>
    <div
      class="relative h-[33px] w-[75px]"
    >
      <img
        alt="zara"
        data-nimg="fill"
        decoding="async"
        fetchpriority="high"
        sizes="75px"
        src="/_next/image?url=%2Fzara.png&w=3840&q=75"
        srcset="/_next/image?url=%2Fzara.png&w=16&q=75 16w, /_next/image?url=%2Fzara.png&w=32&q=75 32w, /_next/image?url=%2Fzara.png&w=48&q=75 48w, /_next/image?url=%2Fzara.png&w=64&q=75 64w, /_next/image?url=%2Fzara.png&w=96&q=75 96w, /_next/image?url=%2Fzara.png&w=128&q=75 128w, /_next/image?url=%2Fzara.png&w=256&q=75 256w, /_next/image?url=%2Fzara.png&w=384&q=75 384w, /_next/image?url=%2Fzara.png&w=640&q=75 640w, /_next/image?url=%2Fzara.png&w=750&q=75 750w, /_next/image?url=%2Fzara.png&w=828&q=75 828w, /_next/image?url=%2Fzara.png&w=1080&q=75 1080w, /_next/image?url=%2Fzara.png&w=1200&q=75 1200w, /_next/image?url=%2Fzara.png&w=1920&q=75 1920w, /_next/image?url=%2Fzara.png&w=2048&q=75 2048w, /_next/image?url=%2Fzara.png&w=3840&q=75 3840w"
        style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
      />
    </div>
    <div
      class="relative h-[33px] w-[166px]"
    >
      <img
        alt="gucci"
        data-nimg="fill"
        decoding="async"
        fetchpriority="high"
        sizes="166px"
        src="/_next/image?url=%2Fgucci.png&w=3840&q=75"
        srcset="/_next/image?url=%2Fgucci.png&w=16&q=75 16w, /_next/image?url=%2Fgucci.png&w=32&q=75 32w, /_next/image?url=%2Fgucci.png&w=48&q=75 48w, /_next/image?url=%2Fgucci.png&w=64&q=75 64w, /_next/image?url=%2Fgucci.png&w=96&q=75 96w, /_next/image?url=%2Fgucci.png&w=128&q=75 128w, /_next/image?url=%2Fgucci.png&w=256&q=75 256w, /_next/image?url=%2Fgucci.png&w=384&q=75 384w, /_next/image?url=%2Fgucci.png&w=640&q=75 640w, /_next/image?url=%2Fgucci.png&w=750&q=75 750w, /_next/image?url=%2Fgucci.png&w=828&q=75 828w, /_next/image?url=%2Fgucci.png&w=1080&q=75 1080w, /_next/image?url=%2Fgucci.png&w=1200&q=75 1200w, /_next/image?url=%2Fgucci.png&w=1920&q=75 1920w, /_next/image?url=%2Fgucci.png&w=2048&q=75 2048w, /_next/image?url=%2Fgucci.png&w=3840&q=75 3840w"
        style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
      />
    </div>
    <div
      class="relative h-[33px] w-[166px]"
    >
      <img
        alt="prada"
        data-nimg="fill"
        decoding="async"
        fetchpriority="high"
        sizes="166px"
        src="/_next/image?url=%2Fprada.png&w=3840&q=75"
        srcset="/_next/image?url=%2Fprada.png&w=16&q=75 16w, /_next/image?url=%2Fprada.png&w=32&q=75 32w, /_next/image?url=%2Fprada.png&w=48&q=75 48w, /_next/image?url=%2Fprada.png&w=64&q=75 64w, /_next/image?url=%2Fprada.png&w=96&q=75 96w, /_next/image?url=%2Fprada.png&w=128&q=75 128w, /_next/image?url=%2Fprada.png&w=256&q=75 256w, /_next/image?url=%2Fprada.png&w=384&q=75 384w, /_next/image?url=%2Fprada.png&w=640&q=75 640w, /_next/image?url=%2Fprada.png&w=750&q=75 750w, /_next/image?url=%2Fprada.png&w=828&q=75 828w, /_next/image?url=%2Fprada.png&w=1080&q=75 1080w, /_next/image?url=%2Fprada.png&w=1200&q=75 1200w, /_next/image?url=%2Fprada.png&w=1920&q=75 1920w, /_next/image?url=%2Fprada.png&w=2048&q=75 2048w, /_next/image?url=%2Fprada.png&w=3840&q=75 3840w"
        style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
      />
    </div>
    <div
      class="relative h-[33px] w-[166px]"
    >
      <img
        alt="calvin"
        data-nimg="fill"
        decoding="async"
        fetchpriority="high"
        sizes="166px"
        src="/_next/image?url=%2Fcalvin.png&w=3840&q=75"
        srcset="/_next/image?url=%2Fcalvin.png&w=16&q=75 16w, /_next/image?url=%2Fcalvin.png&w=32&q=75 32w, /_next/image?url=%2Fcalvin.png&w=48&q=75 48w, /_next/image?url=%2Fcalvin.png&w=64&q=75 64w, /_next/image?url=%2Fcalvin.png&w=96&q=75 96w, /_next/image?url=%2Fcalvin.png&w=128&q=75 128w, /_next/image?url=%2Fcalvin.png&w=256&q=75 256w, /_next/image?url=%2Fcalvin.png&w=384&q=75 384w, /_next/image?url=%2Fcalvin.png&w=640&q=75 640w, /_next/image?url=%2Fcalvin.png&w=750&q=75 750w, /_next/image?url=%2Fcalvin.png&w=828&q=75 828w, /_next/image?url=%2Fcalvin.png&w=1080&q=75 1080w, /_next/image?url=%2Fcalvin.png&w=1200&q=75 1200w, /_next/image?url=%2Fcalvin.png&w=1920&q=75 1920w, /_next/image?url=%2Fcalvin.png&w=2048&q=75 2048w, /_next/image?url=%2Fcalvin.png&w=3840&q=75 3840w"
        style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
      />
    </div>
  </section>
</div>
`;
